app/src/main/java/org/amfoss/templeapp/activities/ReadAllData.java

```diff
@@ -1,138 +1,102 @@
 package org.amfoss.templeapp.activities;
 
 import android.app.ProgressDialog;
+import android.databinding.DataBindingUtil;
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
 import android.view.View;
-import android.widget.Button;
-import android.widget.CheckBox;
-import android.widget.CompoundButton;
-import android.widget.LinearLayout;
-import android.widget.ListView;
 import android.widget.TextView;
 import android.widget.Toast;
 import java.util.ArrayList;
 import org.amfoss.templeapp.R;
 import org.amfoss.templeapp.adapter.MyArrayAdapter;
 import org.amfoss.templeapp.adapter.MyDataModel;
+import org.amfoss.templeapp.databinding.ReadAllBinding;
 import org.amfoss.templeapp.json_api.Controller;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 public class ReadAllData extends AppCompatActivity {
 
-    LinearLayout totalLayout;
-    LinearLayout radioButtonView;
+    ReadAllBinding readAllBinding;
     private int flag;
-    private ListView listView;
     private ArrayList<MyDataModel> list;
     private MyArrayAdapter adapter;
-    private Button readAll;
-    private TextView heading;
-    private CheckBox donatePaid;
-    private CheckBox donateNotPaid;
-    private CheckBox poojaPaid;
-    private CheckBox poojaNotPaid;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.read_all);
 
-        readAll = findViewById(R.id.readAll_btn1);
+        readAllBinding = DataBindingUtil.setContentView(this, R.layout.read_all);
+        readAllBinding.setReadAll(new ReadAll());
+
         list = new ArrayList<>();
         adapter = new MyArrayAdapter(this, list);
-        listView = findViewById(R.id.listView);
-        listView.setAdapter(adapter);
-        heading = findViewById(R.id.heading);
-        radioButtonView = findViewById(R.id.radiogroup_view);
-
-        totalLayout = findViewById(R.id.total_View);
-        donatePaid = findViewById(R.id.donate_paid);
-        donateNotPaid = findViewById(R.id.donate_not_paid);
-        poojaPaid = findViewById(R.id.pooja_paid);
-        poojaNotPaid = findViewById(R.id.pooja_not_paid);
-
-        readAll.setOnClickListener(
-                new View.OnClickListener() {
-                    @Override
-                    public void onClick(View view) {
-                        if (donatePaid.isChecked() && donateNotPaid.isChecked()) {
-                            flag = 5;
-                        }
+        readAllBinding.listView.setAdapter(adapter);
```

Comment:
indentation

Reply:
I think everywhere indentation is correct, I run spotless also for checking It didn't give me any errors